packages/webpack-config/src/index.js

```diff
@@ -125,20 +148,38 @@ const baseConfig = {
 					{
 						loader: 'sass-loader',
 						options: {
-							sourceMap: (mode === 'development'),
+							sourceMap: shouldUseSourceMap,
 							// Set scss debug flag
 							data: `$IS_DEBUG: ${(mode === 'development')};`,
 						},
 					},
 				],
 			},
 
-			// JavaScript - babel-loader transpiles our javascript to ensure browser compatability
 			{
-				test: /\.js$/,
-				exclude: /node_modules/,
-				use: ['babel-loader'],
+				test: /\.(svg|png|jpg|ico|gif|woff|woff2|ttf|eot|doc|pdf|zip|wav|avi|txt)(\?v=\d+\.\d+\.\d+)?$/,
+				oneOf: [
+					// "url" loader works like "file" loader except that it embeds assets
+					// smaller than 10000 bytes
+					{
+						test: [/\.svg$/, /\.gif$/, /\.jpe?g$/, /\.png$/],
+						loader: require.resolve('url-loader'),
```

Comment:
Can we use `loader: 'url-loader'` instead of `loader: require.resolve('url-loader')`? This will make this rule easier to override with webpack-merge.